vector/src/main/java/im/vector/app/features/roomdirectory/RoomDirectoryViewModel.kt

```diff
@@ -188,7 +188,6 @@ class RoomDirectoryViewModel @AssistedInject constructor(
         val mutex = Mutex()
 
         currentJob = viewModelScope.launch {
-<<<<<<< HEAD
             roomDirectories.map { roomDirectoryServer ->
                 val roomDirectoryData = roomDirectoryServer.protocols.first()
```

Comment:
Where coming from this code ?

Reply:
There is a prat comes from Element and i adapt it for Tchap
<img width="889" alt="Capture d’écran 2022-06-10 à 14 26 41" src="https://user-images.githubusercontent.com/15031750/173063960-1aa74f58-ea32-4663-8b0f-07c395dcb7a6.png">

Reply:
@yostyle what do you mean? Perso, I'm wondering what is the reason of the conflict? we change this code a long time ago

Reply:
we had a conflict because the indentation changes

Reply:
Please add tchap comment.